release-content/0.15/migration-guides/12095_Add_features_to_switch_NativeActivity_and_GameActivity_usa.md

```diff
@@ -1,2 +1,46 @@
-Because `cargo-apk` is not compatible with `GameActivity`,  building/running using `cargo apk build/run -p bevy_mobile_example` is no longer possible.
-Users should follow the new workflow described in document.
+`cargo-apk` has been replaced with `cargo-ndk`.
+
+Before:
+
+```shell
+rustup target add aarch64-linux-android armv7-linux-androideabi
+cargo install cargo-apk
+```
+
+After:
+
+```shell
+rustup target add aarch64-linux-android
+cargo install cargo-ndk
+```
+
+Shared object files must be now built for the target architecture before launching package builds
+with the Gradle wrapper.
+
+Before:
+
+```shell
+cargo apk build --package bevy_mobile_example
+```
+
+After:
+
+```shell
+cargo ndk -t arm64-v8a -o android_example/app/src/main/jniLibs build --package bevy_mobile_example
+./android_example/gradlew build
+```
+
+(replace target and project name as required). Note that build output paths have changed. APK builds
+can be found under `app/build/outputs/apk`).
+
+Android Studio may also be used.
+
+Bevy may require the `libc++_shared.so` library to run on Android. This can be manually obtained
+from NDK source, or NDK describes a
+[`build.rs`](https://github.com/bbqsrc/cargo-ndk?tab=readme-ov-file#linking-against-and-copying-libc_sharedso-into-the-relevant-places-in-the-output-directory)
+approach. A suggested solution is also presented in the Bevy mobile example.
+
+Applications that still require `NativeActivity` should:
```

Comment:
There's no mention of NativeActivity being replaced by GameActivity in this file, so the sentence "Applications that still require `NativeActivity` should:" is confusing and missing context.

Reply:
Ah right, I mistakenly assumed these get placed with a title describing the change. Will fix.